ios-trivia-game/ViewControllers/ProfileViewController.swift

```diff
@@ -22,18 +23,21 @@ class ProfileViewController: UIViewController {
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        // self.contentViewCenterY.constant += self.view.bounds.height
+        self.profileLabel.center.x += self.view.bounds.width
         self.contentView.center.x += self.contentView.frame.width
         self.contentView.layoutIfNeeded()
         self.profileImageView.alpha = 0.0
         
-        UIView.animate(withDuration: 0.6, animations: {
-            // self.contentViewCenterY.constant -= self.view.bounds.height
-            
+        UIView.animate(withDuration: 0.8, animations: {
             self.contentView.center.x -= self.contentView.frame.width
             self.profileImageView.alpha = 1
             self.contentView.layoutIfNeeded()
         })
+        
+        UIView.animate(withDuration: 0.7, animations: {
+            self.profileLabel.center.x -= self.view.bounds.width
+            self.profileLabel.layoutIfNeeded()
+        })
     }
     
     override func viewDidLoad() {
```

Comment:
In this method, I think the app is crashing when `self.user.photoUrl` is nil. Can we add guard for this?